### PR TITLE
[scriptassist] change eval by yaml

### PR DESCRIPTION
**Pull request by bw1**
@ailin-nemui

### CODE REVIEW

**Comment by ailin-nemui** [comment]
hi bw1

I know you make lots of work

we must not change the format

sha idea is interesting @dequis do you know if that is useful or already built into https? 

@bw1 what is your reason for coming up with the changes?

**Comment by bw1** [comment]
* format
[eval_test.pl](https://gist.github.com/bw1/946f1a0408064a6e0abc1e5d914e6408)

**Comment by ailin-nemui** [comment]
did you consider yaml?

**Comment by bw1** [comment]
i like yaml. but it's not a [standart modul](https://helpmanual.io/man1/perlmodlib/)

**Comment by ailin-nemui** [comment]
try `CPAN::Meta::YAML`, it's standard module

advantage: we already have _data/scripts.yaml

**Comment by bw1** [comment]
yaml, done

File: _testing/update-scripts-yaml.pl [hunk 1]
```diff
@@ -72,7 +86,14 @@
     }
 } sort keys %newmeta;
 YAML::Tiny::DumpFile('_data/scripts.yaml', \@newdoc);
+YAML::Tiny::DumpFile('scripts.yaml', \@newdoc);
```
Comment (ailin-nemui) [inline _testing/update-scripts-yaml.pl]: just commit a symlink to _data/scripts.yaml

**Comment by bw1** [comment]
done

**Comment by ailin-nemui** [comment]
@bw1 please make a PR **only** with yaml, then I'll merge that quickly

more research is needed about hashing and verification

**Comment by bw1** [comment]
see #642

**Comment by ailin-nemui** [comment]
eval->yaml implemented

**Pull request closed without merging**